src/common/redis_module/ray_redis_module.cc

```diff
@@ -832,8 +832,10 @@ int ResultTableLookup_RedisCommand(RedisModuleCtx *ctx,
 
   RedisModuleString *task_id;
   RedisModuleString *is_put;
+  RedisModuleString *data_size;
+  RedisModuleString *hash;
```

Comment:
`data_size` and `hash` need to be freed down below the same way we free `task_id` and `is_put`, right?